content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -36,10 +36,9 @@ Example: `kubernetes.io/metadata.name=mynamespace`
 
 Used on: Namespaces
 
-When the `NamespaceDefaultLabelName`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled,
-the Kubernetes API server sets this label on all namespaces. The label value is set to
-the name of the namespace.
+The Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}}
```

Comment:
The API server sets this label… that's a meaningful distinction from "control plane" (even though the API server is part of the control plane), since it indicates the label is not set by a controller (and is not set asynchronously)

Reply:
How about, then:
```markdown
The Kubernetes API server (part of the {{< glossary_tooltip text="control plane" term_id="control-plane" >}})
```
?

Reply:
that sgtm

Reply:
```suggestion
The Kubernetes API server (part of the {{< glossary_tooltip text="control plane" term_id="control-plane" >}})
```